File: PythonProject/PhoneCall.py
from datetime import datetime


class PhoneCall:

    def __init__(self, phone_number, start, end):
        self.phone_number = phone_number
        self.start = start
        self.end = end

    def get_start_as_date(self) -> datetime:
        return datetime.strptime(self.start, "%H:%M")

File: CSharpProject/CSharpProject/PhoneCall.cs
namespace CSharpProject;

[Serializable]
public class PhoneCall
{
    public PhoneCall(string phoneNumber, string start, string end)
    {
        PhoneNumber = phoneNumber;
        Start = start;
        End = end;
    }

    private string PhoneNumber { get; set; }
    private string Start { get; set; }
    private string End { get; set; }

    public TimeOnly GetStartAsTime()
    {
        return TimeOnly.Parse(Start);
    }

    public override string ToString()
    {
        return $"PhoneCall=[{PhoneNumber}, {Start}, {End}]";
    }
}